### DISCUSSION TITLE
Delay on translation merges

### DISCUSSION THREAD

**Discussion by undercontr** [medusajs/medusa#6575] [RFC]
I've created a PR for Turkish translation but nobody is taking a look and it's there like a week, can anyone please merge this PR?

[6480](https://github.com/medusajs/medusa/pull/6480)

**Comment by emilselroos** [comment]
I was informed earlier that translation commits will be delayed on purpose because there is some kind of change taking place. Any update on this? Would love to finish my translation asap and get it merged but can't find info on this.

**Comment by souvrard** [comment]
Hi,
Are there some infos about these changes ?
 I'd like to have the admin UI translated in French and I am ready to contribute yet... is it possible ? is there a place where translations are discussed ?

**Comment by MrCasquette** [comment]
Me too, at least two people waiting for the French translations to merge, as there's no way to add a custom file to the project :(

**Comment by emilselroos** [comment]
Admin translations are currently instructed in here:
[https://docs.medusajs.com/learn/resources/contribution-guidelines/admin-translations](https://docs.medusajs.com/learn/resources/contribution-guidelines/admin-translations)
